demo/Dockerfile

```diff
@@ -5,56 +5,22 @@ FROM local/nginx:latest as composer
 RUN --mount=type=cache,target=/root/.composer/cache \
     --mount=id=downloads,type=cache,target=/opt/downloads \
     DOWNLOAD_CACHE_DIRECTORY="/opt/downloads" && \
-    composer create-project drupal/recommended-project:^8.9 \
-        --prefer-dist \
-        --no-interaction \
-        --stability stable \
-        --no-dev \
-        --no-install \
-        /var/www/drupal \
-    && \
+    # Get the Drupal codebase
+    git clone https://github.com/dannylamb/islandora-sandbox && \
```

Comment:
There is a helper https://github.com/Islandora-Devops/isle-buildkit/blob/main/crayfish/Dockerfile#L8 `git-clone-cached.sh` which might be helpful in reducing build times.